docs/getting-started.md

```diff
@@ -2,18 +2,43 @@
 Getting Started with Shunter
 ============================
 
-This guide will teach you how to put together a basic application with Shunter. [API Documentation](usage/index.md) is also available if you need more detail.
-
-Before we begin, you'll need to have [Node.js](https://nodejs.org/) installed, Shunter requires Node.js 4.x–6.x.
+This guide will introduce you to Shunter and teach you how to use it to put together a basic application. [API Documentation](usage/index.md) is also available if you need more detail.
 
 This guide will not explain every feature of Shunter. Its aim is to get you up-and-running in as little time as possible.
 
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
 - [Basics](#basics)
 - [Sample Data](#sample-data)
 - [Templates](#templates)
 - [Styling](#styling)
 - [JavaScript](#javascript)
 
+Introduction
+------------
+
+Looking to create a smart, flexible, and robustly decoupled front end?
+
+Need a hand defining integration points between your front and back end applications?
+
+Want to use the same unified front end across both your rails(/PHP/ASP/other) application, and your Wordpress-driven blog?
```

Comment:
Minor: rails > Rails

Also are parentheses needed here? If so, we could scrap the leading slash